<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_principal</class>
 <widget class="QMainWindow" name="menu_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Games Central</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/joystick_1f579-fe0f.png</normaloff>../img/joystick_1f579-fe0f.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #19171c;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_hangman">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bosch Sans Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(10, 9, 11);
color: white;
border-radius: 10px;
border: 2px dashed rgb(170, 85, 255);
</string>
    </property>
    <property name="text">
     <string>Hangman Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tictactoe">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bosch Sans Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(10, 9, 11);
color: white;
border-radius: 10px;
border: 2px dashed rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Tic Tac Toe</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_games">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bosch Sans Black</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #c0fb09;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>GAMES</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_central">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>central</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/alien-monster_1f47e.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_jokenpo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bosch Sans Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(10, 9, 11);
color: white;
border-radius: 10px;
border: 2px dashed rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Jokenpo</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
